components/General/index.tsx

```diff
@@ -1,24 +1,75 @@
 import { classNames } from '@libs/utils/functions';
 import { Children } from 'libs/types/General';
 import React from 'react';
+import styled from 'styled-components';
 
 type SProps = {
     className?: string;
     label: string;
+    showSectionDetails?: boolean;
 } & Children;
-export const Section: React.FC<SProps> = ({ className, children, label }: SProps) => {
-    return (
-        <div
-            className={classNames(
-                `flex w-full pb-2 last:pb-0 text-sm border-box text-theme-text-secondary`,
-                className ?? '',
-            )}
-        >
-            <div className={'text-left whitespace-nowrap capitalize'}>{label}</div>
-            <span className={'w-full text-right pl-1'}>{children}</span>
-        </div>
-    );
-};
+
+export const Section: React.FC<SProps> = styled(
+    ({ className, children, label, showSectionDetails = false }: SProps) => {
+        return (
+            <div className={classNames(className ?? '')}>
+                <div className={`label ${showSectionDetails ? 'details' : ''}`}>{label}</div>
+                <span className="content">{children}</span>
+            </div>
+        );
+    },
+)`
```

Comment:
noice. Could also make label and content their own styled component. If we ever need to change them through the parent styled components has a really nice way of doing that as well

https://styled-components.com/docs/advanced#referring-to-other-components

Reply:
Still learning what's possible with styled components. I like this feature.